src/main/java/dev/galacticraft/mod/content/block/special/RocketWorkbench.java

```diff
@@ -62,6 +64,17 @@ protected InteractionResult useWithoutItem(BlockState state, Level level, BlockP
         return InteractionResult.CONSUME;
     }
 
+    @Override
+    protected void onRemove(BlockState state, Level level, BlockPos pos, BlockState newState, boolean moved) {
+        if (state.getBlock() != newState.getBlock()) {
+            BlockEntity blockEntity = level.getBlockEntity(pos);
+            if (blockEntity instanceof RocketWorkbenchBlockEntity workbench) {
+                Containers.dropContents(level, pos, workbench.ingredients);
+                Containers.dropContents(level, pos, workbench.chests);
+            }
+        }
+    }
```

Comment:
There needs to be a call to `super.onRemove` here (otherwise the block entity is never deleted).

Reply:
Fixed :+1: